docs/source/classes.rst

```diff
@@ -16,35 +16,33 @@ The subject of a **Pedigree** is represented by an **Individual** class. This cl
    :header-rows: 1
 
    * - Field
-     - Type
      - Multiplicity
      - Description
    * - id
-     - ID
      - 1..1
-     - Logical ID
-   * - name
-     - string
-     - 0..1
-     - Name of the individual
-   * - identifiers
-     - array of ``Identifier``
-     - 0..*
-     - External identifiers for the individual
+     - External identifier for the individual
    * - sex
-     - ``Concept``
      - 1..1
-     - Sex assigned at birth, values: ``male``, ``female``, ``other``, ``unknown``; See `Gender Harmony Context Definitions <https://confluence.hl7.org/display/VOC/Gender+Harmony+Context+Definitions>`_ and `FHIR AdministrativeGender Value Set <http://hl7.org/fhir/ValueSet/administrative-gender>`_.
+     - Sex assigned at birth
    * - gender
-     - ``Concept``
      - 0..1
-     - Presumed or reported gender identity, values: ``male``, ``female``, ``non-binary``, ``non-disclosed``, ``trans``, etc.; See `Gender Harmony Context Definitions <https://confluence.hl7.org/display/VOC/Gender+Harmony+Context+Definitions>`_.
-   * - lifeStatus
-     - ``Concept``
+     - Presumed or reported gender identity
+   * - name
+     - 0..1
+     - Name of the individual
+   * - dateOfBirth
+     - 0..1
+     - Birth date of the individual, can be just birth year in most cases
+   * - age
+     - 0..1
+     - Age of the individual
+   * - raceEthnicityAncestry
      - 0..1
-     - The presumed/accepted life status of the individual as of the pedigree collection date, values: ``alive``, ``deceased``, ``unborn``
+     - Race, Ethnicity, or Ancestry of the individual
```

Comment:
This should be 0..*, and should recommend HANCESTRO

Reply:
@buske, Updated cardinality and description!